capten/config-worker/pkg/activities/git_configure.go

```diff
@@ -36,16 +35,16 @@ func handleGit(ctx context.Context, params model.ConfigureParameters, payload js
 	}
 
 	cred, err := credReader.GetCredential(ctx, credentials.GenericCredentialType,
-		config.VaultEntityName, req.VaultCredIdentifier)

```

Comment:
entity name based on request, it would be "git-project" in this case. Add const for the same